packages/calcite-components/src/components/input-time-zone/input-time-zone.tsx

```diff
@@ -487,6 +493,7 @@ export class InputTimeZone
   componentDidLoad(): void {
     setComponentLoaded(this);
     this.overrideSelectedLabelForRegion(this.open);
+    this.openChanged();
```

Comment:
For this, i'm wondering if its better to call this when there's a ref for the element rather than in componentDidLoad?

I'm not sure it makes a difference in most cases but it just seems more appropriate?

In this case it would be in `setComboboxRef`